src/ui/pages/DownloadPage.tsx

```
@@ -20,18 +20,32 @@ export const DownloadPage = ({ id, files }: DownloadPageProps) => {
         <title>
           <DownloadTitle hasFiles={hasFiles} />
         </title>
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <meta name="description" content="Download your Anki decks from 2anki.net" />
+        <style dangerouslySetInnerHTML={{ __html: `
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)